imageio/plugins/spe.py

```diff
@@ -707,26 +708,32 @@ def _get_data(self, index):
             if index >= self._len:
                 raise IndexError("Image index %i > %i" % (index, self._len))
 
+            frame_size = self._shape[0] * self._shape[1]
```

Comment:
This will produce unexpected/undefined behavior in SPE 3.0 if the file contains differently sized ROIs. (This appears to be one of the shiny new features of SPE 3.0 if I understand correctly.)

Do you have interest/plans for adding support for this, too?

Reply:
I don't plan to work on this, but (per above) I'll make it error out cleanly.

Reply:
That would be much appreciated :)